ci_framework/hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -223,6 +223,9 @@
                     value: ["{{ content_provider_registry_ip }}:5001"]
               {% endif %}
 
+                  - op: add
+                    path: /spec/nodeTemplate/ansible/ansibleVars/edpm_sshd_allowed_ranges
+                    value: ['192.168.122.0/24']
```

Comment:
I think we have this range as a variable at this point - I'd rather not hardcode anything related to 192.168.122.0/24.. @pablintino do you happen to remember where's that range might be fetched from?

Reply:
@bshephar @cjeanner sure, this will do the trick:
`crc_ci_bootstrap_networks_out['compute-0'].default.ip |ansible.utils.ipaddr('network/prefix')  `

Reply:
Let me know if the latest revision addresses your concern here. I just copied Pablo's suggestion into the value section, which I assume is what you were after.

Reply:
all good, thanks @bshephar @pablintino !